### ISSUE TITLE
Reputation template: Redeploy autark client

### ISSUE THREAD

**Issue by stellarmagnet** [AutarkLabs/open-enterprise#1533]
So we can test, we should redeploy beta.autark.xyz using the client repo that has our reputation template.

**Comment by Quazia** [comment]
Actual deployment shouldn't take long, but including some time here to test it's working as well. If there are issues found with the template upon deployment please capture those separately as they don't fall under this issue.

**Comment by ottodevs** [comment]
Done!
The new [beta.autark.xyz](https://beta.autark.xyz) points to [new-template-onboarding](https://github.com/AutarkLabs/aragon/tree/new-template-onboarding) branch (pending PR), based of latest https://github.com/aragon/aragon/commit/c5382349bc201dbcb217c6ac00611035ad02b257

The only thing is we need to renew the certificate, please @rkzel would you take care of that? Also please, if you don't mind configure automatic certificate renewal so we don't need to do it manually each month or two, should we create a new ticket for this task @stellarmagnet ?

**Comment by stellarmagnet** [comment]
@ottodevs, I agree that we should set-up auto renewal of the certificate, I believe we have had this discussion on multiple occasions and I'm not sure why we can't set it up to auto-renew again?
